addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -69,8 +69,8 @@ if (!(_oldCompats isEqualTo [])) then {
 ///////////////
 // check dlls
 ///////////////
-if (toLower (productVersion select 6) in ["linux", "osx"]) then {
-    INFO_2("Operating system does not support DLL file format");
```

Comment:
How did this not get noticed before? and why didn't arma crash at this invalid macro usage? and why did our linter not pick that up?
so many questions.

Reply:
per se linux doesn't support *.dll files, does ist?

Reply:
none of our releases have ever included `.so` files
with this line, people on linux can at least attempt to use all ace features that do not rely on extensions
now it will block access entirely

Reply:
I thought I have seen them on linux server :thinking: